src/lib/main.js

```diff
@@ -37,6 +38,26 @@ cm.Item({
   }
 });
 
+var readerModeKey = Hotkey({
+    combo: "accel-f1",
```

Comment:
use a preference to have it customizable; `combo: sp.prefs.hotkey`
https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/simple-prefs